### PR TITLE
Small tag addition + fixed not using absolute transform

### PR DESCRIPTION
**Pull request by JanSchollaert**
# Description

- Tag function where you can check if an entity has a specific tag.
- Changed using absolute transform when looking if something is in trigger
- Few exceptions added

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] Any dependent changes have been merged and published in downstream modules
- [x] Included 1 or more screenshots
- [x] Code is const correct

### CODE REVIEW

File: include/brickengine/entities/entity_manager.hpp [hunk 1]
```diff
@@ -257,6 +257,9 @@ class EntityManager {
         for (auto& entity : entities_to_delete)
             removeEntity(entity);
     }
+    bool hasTag(int entity, std::string tag) {
+        return tagging_tags.at(tag).count(entity) > 0;
```
Comment (Peter-JanGootzen) [inline include/brickengine/entities/entity_manager.hpp]: You should do
```
if (!tagging_tags.count(tag)) return false
return tagging_tags.at(tag).count(entity) > 0;
```

**Comment by Peter-JanGootzen** [comment]
Hahaha copy pasting my code into the github editor ain't a good idea. :smiling_imp:

**Comment by JanSchollaert** [comment]
I cba to open my laptop lmao

**Comment by JanSchollaert** [comment]
RE-REVIEW THIS

**Pull request merged**